### ISSUE TITLE
Russian-language sources for TV shows are not shown flix trailers .. 

### ISSUE THREAD

**Issue by Whiteripd0** [beatmasterRS/skin.arctic.zephyr.mod#548]
Sorry for bothering you! (I wanted to know if it is possible to return the "Trailer" button in DialogInfo for TV Shows * For the reason that Russian-language sources for TV shows are not shown flix trailers .. If you change the source of tmdb or tvdb to English, then the trailers work, but unfortunately  Russian .. And one more thing (forgive me please!) Is it possible to add the possibility of a movie slogan in bold letters under the movie logo * Maybe an optional function or test?) This is very catchy when choosing a movie in my case .. I would even pay for this function!  Thanks in advance!)

**Comment by beatmasterRS** [comment]
No need to pay for any request or something :) If anyone likes to donate for that, what is done, he can do it.

Do you mean the auto trailer feature or is it releated for all trailer functionality like in e.g. dialog video info? 

In all case, please provide screenshots with extended debugging enabled. You can enable extended debugging in skin settings - Arctic zephyr reloaded

Maybe a video would be better as a screenshots so I can exactly see, what's going on :)

**Comment by Whiteripd0** [comment]
Thank you very much for such a quick response and your help :) I would send donations with great pleasure, but due to the fact that I do not live abroad, unfortunately (My donation would be considered a trifle because of the exchange rate .. But I will definitely make a donation anyway as soon as possible) I apologize in advance that perhaps my donation will not be the same as others send .. As for my problem, I will send you screenshots and maybe even from the screenshots you will understand my problem more clearly) If the screenshots do not help, then I will try to send you video as soon as possible.
![Uploading Screenshot_5.png…]()
This screenshot shows that the "Trailer" button is missing and so for all TV shows after the update from 1.58 to 1.59.

And the video shows that for 20 seconds the trailer also does not turn on (
https://streamable.com/410atd
If you change the source of tvdb to English, then the trailers work fine without problems :) But unfortunately I do not live abroad and do not know English very well .. I would like to be able to watch trailers in my language, or if it's not so easy then return the `` Trailer '' button in DialogVideoInfo.) It was in 1.58 and in 1.59 it disappeared for all TV shows (Forgive me for writing so much, maybe you are now confused in my writing)) I just wanted how can I accurately describe the problem that I currently have

**Comment by beatmasterRS** [comment]
Coukd you please enable extended Debugging before making a video so that I can see the debugging infos if the auto trailer feature?

**Comment by Whiteripd0** [comment]
Sorry! I completely forgot about it..
https://streamable.com/9ek5cl

**Comment by beatmasterRS** [comment]
It seems, that listitem.trailer is empty, that means that maybe the addon nor the scraper doesn't provide a trailer link...

In dialog video info is there a trailer button visible for that movie?

**Comment by Whiteripd0** [comment]
There is no such button at the moment..But it was in the last version) And it worked perfectly even for a Russian source. But I downloaded the files from torrents. And trailers automatically work only if tmdb (tvdb) puts preferred language english instead of rus (Is it possible to somehow return that button? Please tell me?) Or maybe there is an opportunity to make a trailer for Russian sources as well?

**Comment by Whiteripd0** [comment]
For movies there is,but for TV shows this button is missing everywhere unfortunately 
If in short*)

**Comment by beatmasterRS** [comment]
Please read carefully, what I had asked for... Is there a trailer-button for the selected movie in dialog video info, when the trailer does not play automatically? If yes, please make a video of that behaviour, thanks :)

**Comment by Whiteripd0** [comment]
I'm sorry!) Here is the video. The video shows that the "Trailer" button is missing
https://streamable.com/zu69qe

**Comment by beatmasterRS** [comment]
Okay then it's not skin releated, cause the trailer button will only be visible, if the listitem.trailer infolabel is filled. Except when you have enabled IMDb trailers option... 

If the listitem.trailer infolabel is filled, the trailer button is visible and if auto trailers is enabled, the trailer starts after the selected delay automatically.

**Comment by Whiteripd0** [comment]
I have never turned on the imdb function before, but I also have most of the media icons in the furniture disabled, but I do not know if this is because of this.. And I understood the trailer system itself) It turns out that the listitem is not empty and then everything works well.
BY THE WAY! The trailer button is available in the TV show library and in absolutely all TV shows!) I probably haven't visited the library for so long..)) But on the home page in the widgets in DialogVideoInfo, this button is missing everywhere(

**Comment by Whiteripd0** [comment]
and again please excuse me!  I was wrong again .. There are no trailer buttons in the library either .. It is only when scraper change the preferred language from Russian to English and then this button is in the library.  But in versions 1.58, this button was absolutely everywhere.  And it worked just fine!) Tell me please, can you somehow restore the functionality of this button?  I could roll back to 1.58, but then I won't be able to update anymore .. I don't know how to be now (

**Comment by beatmasterRS** [comment]
Ah okay, it could be that, there was bingie.helper support, that i had removed for tv show trailers. Please test in 1.1.58 and do the following:

- open dialog video info
- focus the trailer button
- make a screenshot

Please post the screenshot here, so i can see, which button id the trailer button has had. Thanks

**Comment by Whiteripd0** [comment]
![Screenshot_12](https://user-images.githubusercontent.com/91340448/134923681-4c2b3a6e-96f6-46eb-b800-a8147adf74a3.png)

**Comment by beatmasterRS** [comment]
Thanks for your help! I will try to readd support für bingie.helper script soon :)

**Comment by Whiteripd0** [comment]
Thank you very much too :) I will hope for the best ..) I really like this opportunity and I will hope that something can be done) Now I use it with no other spins except Zephyr2 and use others just for the test.  This is the most aesthetically neat skin that screams elegance!  :) Thank you so much again :)

**Comment by beatmasterRS** [comment]
Please test:

https://github.com/beatmasterRS/skin.arctic.zephyr.mod/archive/refs/heads/trailer-patch-1.zip

**Comment by Whiteripd0** [comment]
Hello again :) Now it works just as great and perfect as in the previous version) THANK YOU VERY MUCH !!!) I am infinitely grateful to you!  I can't believe it works again😭

**Comment by Whiteripd0** [comment]
Thank you so much for such a quick help😊 And also thank you very much for the fact that you didn’t refuse to help with my problem😅

**Comment by Whiteripd0** [comment]
I am very sorry for bothering you so often! (Thank you very much again for solving the problems with the trailers😃 Yesterday there was simply no limit to joy! And today, of course, too ..)😌But I still have one problem that I cannot solve, yesterday I tried  to do it but nothing helped.  This is a problem with "discart" This works in DialogVideoInfo, but there are no playbacks in the playback itself .. (I turned this function on and off, changed the settings, re-installed discart, changed the player views,) Unfortunately, nothing helped .. Sorry for the trouble!(
